addons/sourcemod/translations/surftimer.phrases.txt

```diff
@@ -535,7 +535,7 @@
 	"Commands8"
 	{
 		"#format"	"{1:s}"
-		"en"		"{1} How did you hit 1024 save locs?"
+		"en"		"{1} How did you hit 128 save locs?"
```

Comment:
I agree with the sentiment - while we are correcting the figure, a proper, less silly message would be preferable. Something along the lines of "Maximum of 128 savelocs created" perhaps.

Reply:
I changed the message, lmk if it's alright.